src/sequelize-graphql-schema.js

```diff
@@ -579,9 +870,9 @@ const generateModelTypes = (models, remoteTypes) => {
       const cache = {};
```

Comment:
move `const cache = {}` to outside for loop `for (let modelName in models) {` to avoid enum caching bug..

Reply:
working on